File: 332/2A.pl
#!/usr/bin/perl

sub min {
    return $_[0] < $_[1] ? $_[0]: $_[1];
}

sub max {
    return $_[0] > $_[1] ? $_[0]: $_[1];
}


@d = split(' ', <STDIN>);
$max_one = max(max($d[0], $d[1]), $d[2]);
$ans = min($d[0] + $d[1] + $d[2], 2 * ($d[1] + $d[2] + $d[0] - $max_one));
print $ans, "\n";

File: 330/2A.py
#!/usr/bin/python

import sys

f = sys.stdin
n, m = f.readline().strip().split(' ')
n = int(n)
m = int(m)
ans = 0
for it in xrange(n):
    arr = [int(x) for x in f.readline().strip().split(' ')]
    for sit in xrange(m):
        if arr[2 * sit] + arr[2 * sit + 1] > 0:
            ans += 1
print ans

